tests-unit/Client/RequestTimeoutMiddlewareTest.php

```diff
@@ -12,7 +12,7 @@ class RequestTimeoutMiddlewareTest extends TestCase
     /**
      * @return void
      */
-    public function testWillSetDefaultTimeout()
+    public function testWillSetDefaultTimeout(): void
```

Comment:
tady zustaly anotace `@return void`

Reply:
Takových věcí je tam všude milion a teď by bylo epické to fixovat. To se má fixnout automaticky přes PHPCS - tak jsem to zkusil v https://github.com/keboola/storage-api-php-client/pull/802